Source: John
Upstream: No
Reason: Conflicting declaration of copy_file_range on GLIBC

FAILED: e2fsprogs/misc/create_inode.c.o 
cc -std=gnu11 -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe     -Ie2fsprogs/lib -Ie2fsprogs/lib/ext2fs -Icore/libsparse/include -c e2fsprogs/misc/create_inode.c -o e2fsprogs/misc/create_inode.c.o
e2fsprogs/misc/create_inode.c:406:18: error: conflicting types for 'copy_file_range'
 static errcode_t copy_file_range(ext2_filsys fs, int fd, ext2_file_t e2_file,
                  ^~~~~~~~~~~~~~~
In file included from e2fsprogs/misc/create_inode.c:20:
/usr/include/unistd.h:1107:9: note: previous declaration of 'copy_file_range' was here
 ssize_t copy_file_range (int __infd, __off64_t *__pinoff,
         ^~~~~~~~~~~~~~~

--- e2fsprogs/misc/create_inode.c	2019-01-02 20:07:51.685424116 +0100
+++ -	2019-01-02 20:08:17.379127932 +0100
@@ -17,6 +17,7 @@
 #include <time.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#undef __USE_GNU
 #include <unistd.h>
 #include <limits.h> /* for PATH_MAX */
 #ifdef HAVE_ATTR_XATTR_H
